docs/MVS_Outputs.rst

```diff
@@ -100,21 +100,100 @@ Technical data
+
+The renewable share of local generation describes how much of the energy generated locally is produced from renewable sources.
+It does not take into account the consumption from energy providers.
+
+The renewable share of local generation for each sector does not utilize energy carrier weighting but has a limited, single-vector view:
+
+.. math::
+        REGen_v &=\frac{\sum_i {E_{rgen,i}}}{\sum_j {E_{gen,j}}}
+
+        \text{with } v &\text{: Energy vector}
+
+        rgen &\text{: Renewable generation}
+
+        i &\text{: Renewable generation assets 1,2,…}
+
+        gen &\text{: Renewable and non-renewable generation}
+
+        j &\text{: Renewable and non-renewable generation assets 1,2,…}
+
+For the system-wide share of local renewable generation, energy carrier weighting is used:
+
+.. math::
+        REGen &=\frac{\sum_i {E_{rgen,i} \cdot w_i}}{\sum_j {E_{gen,j} \cdot w_j}}
+
+        \text{with } rgen &\text{: Renewable generation}
+
+        i &\text{: Renewable generation assets 1,2,…}
+
+        gen &\text{: Renewable and non-renewable generation}
+
+        j &\text{: Renewable and non-renewable generation assets 1,2,…}
+
+        w &\text{: Energy carrier weighting factor for output of asset i/j}
+
+
+:Example:
+
+An energy system is composed of a heat and electricity side. Following are the energy flows:
+
+* 100 kWh from a local PV plant
+* 0 kWh local generation for the heat side
+
+This results in:
+
+* A single-vector renewable share of local generation of 0% for the heat sector.
+* A single-vector renewable share of local generation of 100% for the electricity sector.
+* A system-wide renewable share of local generation of 100%.
+
+
+.. _kpi_renewable_factor:
+
+Renewable factor (RF)
 #####################
 
-Describes the share of the MES demand that is supplied from renewable sources.
+Describes the share of the energy influx to the local energy system that is provided from renewable sources.
+This includes both local generation as well as consumption from energy providers.
 
 .. math::
-        RES &=\frac{\sum_i {E_{RES,generation} (i) \cdot w_i}}{\sum_j {E_{generation}(j) \cdot w_j}+\sum_k {E_{grid} (k)}}
+        RF &=\frac{\sum_i {E_{rgen,i} \cdot w_i + RES \cdot E_{grid}}}{\sum_j {E_{gen,j} \cdot w_j}+\sum_k {E_{grid} (k)}}
 
-        \text{with } &i \epsilon \text{[PV,Geothermal,…]}
+        \text{with } rgen &\text{: Renewable generation}
 
-        &j \epsilon \text{[generation assets 1,2,…]}
+        i &\text{: Renewable generation assets 1,2,…}
 
-        &k \epsilon \text{[DSO 1,2…]}
+        gen &\text{: Renewable and non-renewable generation}
 
-** The content of this section was copied from the conference paper handed in to CIRED 2020**
+        j &\text{: Renewable and non-renewable generation assets 1,2,…}
+
+        RES &\text{: Renewable energy share of energy provider}
+
+        k &\text{: Energy provider 1,2…}
+
+        w &\text{: Energy carrier weighting factor for output of asset i/j/k}
```

Comment:
same comment as above, moreover I don't see `w_k` being used in the equation

Reply:
84c36574083504810c646af8ed5488df5deab166